src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -18,6 +18,7 @@ import scala.util.parsing.json.JSONObject
 
 final case class RelationConfig(
     auth: Auth,
+    clientTag: Option[String],
```

Comment:
Let's make it possible to set the application name as well.

Reply:
So make spark 'invisible' in terms of these headers?

Reply:
If the user overrides it with something unrelated to Spark: yes. It's more useful to have "JoelWeatherTransform" as an identifier if that's what I choose to override it with - of course I could choose "JoelWeatherTransform-Cognite-Spark-1.4.6" as well.